src/components/Footer.astro

```diff
@@ -8,21 +8,36 @@ const { styling } = Astro.props
   <div
     class={classNames(
       'container mx-auto flex w-full flex-col items-center justify-between gap-8 border-t px-5 py-14 md:flex-row md:gap-0 md:px-10 lg:px-12',
-      `${styling ? 'text-black' : 'text-white'} `,
+      `${styling ? 'text-black' : 'text-white'}`,
+      `${styling ? 'border-black' : 'border-white'}`,
     )}
   >
-    <a class="text-lg font-bold leading-7 tracking-wide md:text-xl lg:text-2xl lg:leading-7" href="mailto:aleksandra@alexuix.design"
-      >aleksandra@alexuix.design</a
+    <a
```

Comment:
All these `<a>` should be a component to which you forward props of `label` and `href`.

Also, technically, if you are displaying a list of items it should be in a `ul` `li`, also SEO prefers that in components like `nav`, `footer`, `header` etc.